openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1168,9 +1170,19 @@ A Credential Issuer is identified by a case sensitive URL using the `https` sche
 
 The Credential Issuer's configuration can be retrieved using the Credential Issuer Identifier.
 
-Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`. 
+Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`.
+
+To fetch the Credential Issuer Metadata, a requester MUST send a HTTP GET to the path formed following the steps above utilizing TLS. The Credential Issuer MUST return a JSON document compliant with this specification using the `application/json` media type and a HTTP Status Code 200.
+
+The Wallet is RECOMMENDED to send an Accept-Language Header in the HTTP GET request to indicate the particular languages he is interested in. The Credential Issuer is RECOMMENDED to send a subset of internationalized display data containing the requested languages and using the HTTP Content-Language Header. The Credential Issuer MAY ignore the Accept-Language Header and send all supported languages or a any chosen default subset.
```

Comment:
Small improvements to wording:

```suggestion
The Wallet is RECOMMENDED to send an `Accept-Language` Header in the HTTP GET request to indicate the language(s) preferred for display. The Credential Issuer is RECOMMENDED to send a subset of internationalized display data containing the requested languages and using the HTTP `Content-Language` Header. The Credential Issuer MAY ignore the `Accept-Language` Header and send all supported languages or an any chosen default subset.
```

I'm not sure about the 'MAY' here. It seems like saying you MAY ignore the Accept-Language header conflicts with the previous sentence that says it's RECOMMENDED to process it.

Reply:
> and using the HTTP `Content-Language` Header
- The display entries already contain `locale` information, so the `Content-Language` header on the response is not strictly needed, IMO.
- Perhaps we should add that if the `Content-Language` header is present, it should be compatible with the locales used inside the metadata (`Content-Language` can have more than one language).

Reply:
@jogu Do you mean RECOMMENDED and MAY oppose each other? So then whats our opinion here? Issuers are RECOMMENDED OR MAY send the according subset?

Reply:
@pmhsfelix thats true. But its also only RECOMMENDED. What do you propose?

Reply:
> @jogu Do you mean RECOMMENDED and MAY oppose each other? 

Yes, exactly.

> So then whats our opinion here? Issuers are RECOMMENDED OR MAY send the according subset?

I'm not actually sure. I think the real position is that it's probably only worth the complexity of doing this if your metadata is pretty large, e.g. over 30 kilobytes with compression applied? (Or put another way, I'm not sure there's much point in doing this kind of optimisations until the data is notably larger than a single fairly typical photo thumbnail on a website) - so probably for many issuers it'd probably not be worth the effort. So I guess either "MAY" or perhaps "RECOMMENDED if the metadata size exceeds XXX"?

Reply:
agreed on the WG call that the spirit is `MAY`. @Sakurann to make a suggestion and @paulbastian @jogu to help bikeshed.